app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    unmasked_count = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
```

Comment:
better to use NotVaccinatedError

Reply:
According to the paragraph "Notes" in the "README.md" file, the VaccineError exception should be used to catch both NotVaccinatedError and OutdatedVaccineError that could be thrown by the method "visit_cafe", as per project task. Also, it seems the tests of this task imply this kind of logic otherwise they won't be passed, unless an extra "except" clause will be added to catch OutdatedVaccineError with the same return message which seems to be redundant and doesn't comply with the stated requirements.